### DISCUSSION TITLE
EOL of Superset

### DISCUSSION THREAD

**Discussion by makiyt-nttd** [apache/superset#18975] [Q&A / Help]
Hi, Community
I would like to know the End-of-Life for each version of Superset.
What is the policy for EOL if there is no clearly defined date?
Thanks

**Comment by donatelloOo** [comment]
You can check this discussion, if still maintained: https://github.com/apache/superset/discussions/24813
They finally refers to this URL: https://github.com/apache/superset/wiki/Release-Process#schedule

**Reply by btmakiyt** [reply]
Thank you for letting me know! I appreciate it.
